lib/Thingiverse.pm

```diff
@@ -60,8 +52,19 @@ our $client_id          = 'c587f0f2ee04adbe719b';
 our $access_token       = 'b053a0798c50a84fbb80e66e51bba9c4';
```

Comment:
Please note, I'm working here:
https://github.com/perlygatekeeper/perlythingkeeper/pull/1/files
Don't merge this man.
Paul • 2:26 PM
Paul Salcido
OK, Steve, I think I have my first pull request ready to go.  All tests are now passing and I'm happy with most of the work.  After that I have plans for two more pull requests:  Documentation and Dependency Injection.  These will be important for a SOLID code base.  Regardless, will talk later.
Paul • 4:32 PM